cyclegan_arch/mnist_to_svhn/solver.py

```diff
@@ -231,6 +235,160 @@ def set_expectation_and_std(self):
 
     def as_np(self, data):
         return data.cpu().data.numpy()
+    
+    def test(self,svhn_test_loader,mnist_test_loader):
+        svhn_test_iter = iter(svhn_test_loader)
```

Comment:
The new test function.
Copied mostly from the train function.
No grad updates and uses test loaders.
Calculates all the same losses.